ios/FluentUI/Popup Menu/PopupMenuItemCell.swift

```diff
@@ -111,6 +111,13 @@ class PopupMenuItemCell: TableViewCell, PopupMenuItemTemplateCell {
         }
         updateColors()        // until popupmenuitemcell actually supports token system, clients will override colors via cell's backgroundColor property
         backgroundStyleType = .custom
+
+        tokenSetSink = tokenSet.sinkChanges { [weak self] in
+            guard let strongSelf = self else {
+                return
+            }
+            strongSelf.updateAppearance()
```

Comment:
We need to call updateColors after updateAppearance or it wouldn't fix the TVC overriding PopupMenu colors

Reply:
Fixed, confirmed with testing